repl/src/main/resources/fake_shell.py

```diff
@@ -403,6 +406,9 @@ def magic_table(name):
         cols = []
         data.append(cols)
 
+        if os.environ.get("LIVY_TEST") != "true" and isinstance(row, pyspark.sql.types.Row):
```

Comment:
when os.environ.get("LIVY_TEST") == "true"
fake_shell do not import spark library (can not find pyspark.sql.types.Row)